server/schema/questions/addEditor.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  required: ['editor'],
+  properties: {
+    editors: {
+      type: 'string',
+    },
+  },
+}
```

Comment:
```suggestion
const schema = {
  required: ['editor'],
  properties: {
    editor: {
      type: 'string',
    },
  },
  errorMessage: {
    required: {
      editor: 'Editor required',
    },
    properties: {
      editor: 'Invalid editor',
    },
    _: 'Invalid data',
  },
}

module.exports = schema

```

in order to catch the error of sending an array of editors instead of a single name via frontend

Reply:
Error message will be automatically set by ajv, like invalid field "field_name"..I think this is not required